staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/validation/validation.go

```diff
@@ -33,7 +33,7 @@ import (
 func ValidateAPIService(apiService *apiregistration.APIService) field.ErrorList {
 	requiredName := apiService.Spec.Version + "." + apiService.Spec.Group
 
-	allErrs := validation.ValidateObjectMeta(&apiService.ObjectMeta, false, false,
+	allErrs := validation.ValidateObjectMeta(&apiService.ObjectMeta, true, false,
```

Comment:
why the reversed boolean?

Reply:
It indicates the scope is changed from cluster-scoped to tenant-scoped. True means that the validator will check to make sure that "Tenant" in ObjectMeta is not empty.